src/components/adaptive-cards/InputChoiceSet/InputChoiceSet.jsx

```diff
@@ -0,0 +1,123 @@
+import React, {useContext, useEffect} from 'react';
+import PropTypes from 'prop-types';
+import webexComponentClasses from '../../helpers';
+import {acPropTypes, registerComponent} from '../Component/Component';
+import AdaptiveCardContext from '../context/adaptive-card-context';
+import {Select} from '../../generic';
+import Checkbox from '../../inputs/Checkbox/Checkbox';
+import Label from '../../inputs/Label/Label';
+import RadioButton from '../../inputs/RadioButton/RadioButton';
+
+/**
+ * Adaptive Cards InputChoiceSet component
+ * https://adaptivecards.io/explorer/Input.ChoiceSet.html
+ *
+ * @param {object} props  React props passed to the component
+ * @param {object} props.data  Active cards definition
+ * @param {string} [props.className]  Custom CSS class to apply
+ * @param {object} [props.style]  Custom style to apply
+ * @returns {object} JSX of the component
+ */
+export default function InputChoiceSet({
+  data, className, style,
+}) {
+  const [cssClasses] = webexComponentClasses('adaptive-cards-input-choice-set', className);
+  const {setValue, getValue, setInput} = useContext(AdaptiveCardContext);
+  const dataValue = getValue(data.id);
+  const values = Object.fromEntries(dataValue.split(',').map((v) => [v, true]));
```

Comment:
```js
dataValue ? Object.fromEntries(String(dataValue).split...) : {};
```
otherwise it will create a `{'': true}` object if value is empty string or an error if the value is undefined or not a string